pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -126,12 +126,6 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, pa *autoscalingv1alpha1.
 	mode := nv1alpha1.SKSOperationModeProxy
 
 	switch {
-	// When activator CA is enabled, force activator always in path.
-	// TODO: This is a temporary state and to be fixed.
-	// See also issues/11906 and issues/12797.
-	case len(config.FromContext(ctx).Network.ActivatorCA) > 0:
-		mode = nv1alpha1.SKSOperationModeProxy
-
```

Comment:
Why are we removing this right now?

I don't think all the `net-*` implementations are ready to handle dynamic certs between `activator` and `queue-proxy`.

Reply:
As I mentioned above, I'd prefer to keep activator-always-in-path until we can do SNI, e.g. https://www.agwa.name/blog/post/writing_an_sni_proxy_in_go